frontend/src/component/changeRequest/ChangeRequest/Changes/Change/ConflictWarning.tsx

```diff
@@ -0,0 +1,24 @@
+import { Alert } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+export const ConflictWarning: React.FC<{
```

Comment:
Wonder whether this should be renamed into "ConditionalConflictWarning" or something? wdyt?

Reply:
Yeah, that was my thinking as well. It feels like a decent tradeoff between encapsulating the component and giving control to the caller.